setup.php

```diff
@@ -1,5 +1,10 @@
 <?php
-if (!empty($_POST['checkfront_host'])) {
+if (!empty($_POST['checkfront_host']) && !empty($_POST['_wpnonce'])) {
+	//prevent CSRF attacks
+	if (!wp_verify_nonce($_POST['_wpnonce'], 'update_checkfront_host')) {
```

Comment:
Should we be logging anything here?

Reply:
Yes, probably. However, we don't have logging set up for the Wordpress Plugin, so the logs wouldn't go anywhere. We could ask DevOps to setup logging for this repo, but I'm not sure how worth it that would be